.babelrc

```diff
@@ -1,7 +1,3 @@
 {
-  "plugins": [
-    "transform-es2015-parameters",
-    "babel-plugin-transform-es2015-spread",
-    "transform-es2015-modules-commonjs"
-  ]
+  "presets": ["es2015-node4"]
```

Comment:
Would be a good idea to use the shiny new official babel preset. https://www.npmjs.com/package/babel-preset-env

With babel-preset-env you can do this awesome thing: ` "presets": [["env", {"targets": {"node": "current"}}]]`